Source/Core/Core/HotkeyManager.cpp

```diff
@@ -246,6 +285,7 @@ void Initialize()
 
   // load the saved controller config
   s_config.LoadConfig(true);
```

Comment:
Looks like you could change these 2 lines to a call to `LoadConfig`.

Reply:
Oh yeah, silly me.